### PR TITLE
@feature/comments

### PR DESCRIPTION
**Pull request by ukirderohit**

### CODE REVIEW

File: packages/im-base/src/services/http.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
```
Comment (pranay2401) [inline packages/im-base/src/services/http.js:1]: We don't need this file as we are directly calling from youtubeService.js

File: packages/im-base/src/services/helpers/http.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { BASE_HEADERS, BASE_PARAMS, BASE_URL } from "../constants/modules.constants"
```
Comment (pranay2401) [inline packages/im-base/src/services/helpers/http.js:1]: We don't need this file as we are directly calling from youtubeService.js

File: packages/im-base/src/utils/getDataFromResponseSingle.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Lodash
+import _get from "lodash/get";
```
Comment (pranay2401) [inline packages/im-base/src/utils/getDataFromResponseSingle.js]: name can be better

File: packages/im-components/src/molecules/collapsibleContent/CollapsibleContent.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+CollapsibleContent.propTypes = {
+  text: PropTypes.string,
+  limit: PropTypes.number,
+  disabled: PropTypes.bool,
+};
+
+CollapsibleContent.defaultProps = {
+  text: "",
```
Comment (pranay2401) [inline packages/im-components/src/molecules/collapsibleContent/CollapsibleContent.js]: use EMPTY_STRING object

File: packages/im-ott/src/organisms/comments/components/allComments/AllComments.js [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+AllComments.propTypes = {
+  videoId: PropTypes.string.isRequired,
+  onClose: PropTypes.func,
+  visible: PropTypes.bool,
+};
+
+AllComments.defaultProps = {
+  onClose: _noop,
+  visible: false,
```
Comment (pranay2401) [inline packages/im-ott/src/organisms/comments/components/allComments/AllComments.js:135]: defaultProp for videoId?

File: packages/im-ott/src/organisms/comments/components/allComments/allComments.module.scss [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      margin-right: 2rem;
+    }
+
+    .postCommentButton {
+      font-size: 2rem;
+    }
+  }
+
+  .commentsList {
+    height: calc(100% - 56px);
```
Comment (pranay2401) [inline packages/im-ott/src/organisms/comments/components/allComments/allComments.module.scss]: px to rem

**Review by pranay2401 — CHANGES_REQUESTED** [review]

File: packages/im-base/src/services/helpers/http.js (hunk 2 above)
Comment (ukirderohit) [reply packages/im-base/src/services/helpers/http.js:1]: it's using base params as helper, we need that

File: packages/im-ott/src/organisms/comments/components/allComments/AllComments.js (hunk 5 above)
Comment (ukirderohit) [reply packages/im-ott/src/organisms/comments/components/allComments/AllComments.js:135]: doesn't require as its forwarded thru parent prop

**Review by pranay2401 — APPROVED** [review]

**Pull request merged**